airflow/providers/amazon/aws/operators/redshift_to_s3.py

```diff
@@ -72,10 +75,11 @@ class RedshiftToS3Transfer(BaseOperator):
     @apply_defaults
     def __init__(  # pylint: disable=too-many-arguments
             self,
-            schema: str,
-            table: str,
             s3_bucket: str,
             s3_key: str,
+            schema: str = None,
```

Comment:
This is a breaking change so it should be described in UPDATING.md. It's ok for 2.0 (we allow such backwards incompatibilities) but it must be described there.

Reply:
I see! I will document it in UPDATING.md